src/grpc.ts

```diff
@@ -49,6 +52,34 @@ export interface MetadataValue {
   equals: Function;
 }
 
+/*
+ * Async version of readFile.
+ *
+ * @returns {Promise} Contents of file at path.
+ */
+async function readFileAsync(path: string): Promise<string> {
```

Comment:
We'll need to replace this with `fs.promises.readFile` after we drop Node 10 :)

Reply:
example code:
```
const fs = require('fs');
const path = require('path');

const filepath = path.resolve(__dirname, 'file.txt');

(async () => {
  const data = await fs.promises.readFile(filepath, 'utf8');
  console.log(data);
})();
```

Reply:
would you like me to add a TODO?

Reply:
nope, this is just a side note. I don't believe in TODOs anyway :)